public/_headers

```diff
@@ -7,3 +7,8 @@
 
 /*
   Content-Security-Policy: CSP-DIRECTIVES-PLACEHOLDER;
+  X-frame-options: sameorigin
+  X-XSS-Protection: 1; mode=block
+  X-Content-Type-Options: nosniff
+  Referrer-Policy: strict-origin-when-cross-origin
+  Feature-Policy:
```

Comment:
Netlify seems to prune empty headers:

<img width="1208" alt="missing header" src="https://user-images.githubusercontent.com/86275/44851420-8ac7fc00-ac60-11e8-8031-796fc2aec283.png">

Then let us just remove the "Feature-Policy" header for now.

As seen on https://securityheaders.com/?q=https%3A%2F%2Fdeploy-preview-104--scrivito-example.netlify.com&hide=on&followRedirects=on